lib/api/goal/common/PushImpact.ts

```diff
@@ -40,8 +40,7 @@ export class PushImpact extends FulfillableGoalWithRegistrations<PushImpactListe
                 ...dependsOn: Goal[]) {
 
         super({
-            ...PushImpactDefinition,
-            ...getGoalDefinitionFrom(goalDetailsOrUniqueName, DefaultGoalNameGenerator.generateName("push-impact")),
+            ...getGoalDefinitionFrom(goalDetailsOrUniqueName, DefaultGoalNameGenerator.generateName("push-impact"), PushImpactDefinition),
```

Comment:
Is that definition still being used somewhere? Otherwise: delete.

Reply:
same as above.